var/spack/repos/builtin/packages/qmcpack/package.py

```diff
@@ -141,17 +135,15 @@ class Qmcpack(CMakePackage, CudaPackage):
     depends_on('py-matplotlib', when='+gui', type='run')
 
     # B-spline basis calculation require a patched version of
-    # Quantum Espresso 6.4 (see QMCPACK manual)
```

Comment:
Is it possible to keep 6.4 in addition to 6.4.1? We try not to remove old versions of packages for reproducibility.

Reply:
You have raised this issue in the past, but we don't maintain old QE converters and have a tendency to make the old ones disappear :)

Reply:
Thanks, you'll prob have to remind me each time, sorry for being a broken record 😛